src/components/map/toolbar/EventLog.tsx

```diff
@@ -14,16 +19,86 @@ interface IEventLogProps {
 
 @observer
 export default class EventLog extends React.Component<IEventLogProps, IEventLogState> {
+    scrollRef: React.RefObject<HTMLDivElement>;
+
     constructor(props: IEventLogProps) {
         super(props);
         this.state = {
             isOpen: false,
             isAlertShown: false,
         };
+        this.scrollRef = React.createRef();
+    }
+
+    private getLogRow = (action: logActions | null, objectId: string, entity: entityNames) => {
```

Comment:
What is our practice for a method that returns JSX? I think it doesn't start with get

Reply:
I don't find anything suggesting that we have a practise for this in our contributions file: 
https://github.com/HSLdevcom/jore-map-ui/blob/master/Contributions.md